CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ endif()
 include(CMakeDependentOption)
 cmake_dependent_option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY
   "Build with file/folder reorg backward compatibility enabled" ON "NOT WIN32" OFF)
-if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY AND NOT WIN32)
```

Comment:
This change is redundant. The backwards-compatibility headers are not being generated on Windows.

Reply:
You're correct, thanks. These PRs were autogenerated, and the method didn't detect the `cmake_dependent_option`. I'll close this and the rocSOLVER PR.